addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -240,6 +240,9 @@ $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o
 
 // Input
 $input-bg: $o-white !default;
+$form-check-input-border-radius: 0 !default;
```

Comment:
this should be in enterprise I guess (if needed there)

Reply:
Indeed 👍, I divided it in a new PR : https://github.com/odoo/enterprise/pull/31092

Reply:
... but why is still here?

Reply:
Sorry, I forgot to push the community branch... 🙈  it's done 👍